clients/java/signalr/src/main/java/com/microsoft/signalr/HubConnection.java

```diff
@@ -416,6 +438,14 @@ public void send(String method, Object... args) {
         sendHubMessage(invocationMessage);
     }
 
+    /**
+     * Invokes a hub method on the server using the specified name and arguments.
```

Comment:
Suggest: "Invokes a hub method on the server using the specified **method** name and arguments."

(To align with [.NET equivalent](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.signalr.client.hubconnectionextensions.invokeasync?view=aspnetcore-2.1#Microsoft_AspNetCore_SignalR_Client_HubConnectionExtensions_InvokeAsync_Microsoft_AspNetCore_SignalR_Client_HubConnection_System_String_System_Object_System_Object_System_Object_System_Object_System_Object_System_Object_System_Object_System_Object_System_Object_System_Object_System_Threading_CancellationToken_) docs, for consistency)